apps/channels/admin.py

```diff
@@ -30,7 +31,3 @@ def team(self, obj):
 
     def get_changeform_initial_data(self, request: HttpRequest) -> Dict[str, str]:
         return {"extra_data": {"bot_token": "your token here"}}
```

Comment:
Out of curiosity, what was the thinking of including them again?

Reply:
This could actually be removed now